noise.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+    <link href="noise.css" rel="stylesheet" />
+  </head>
+  <body>
+    <button id="record">Record</button>
+    <button id="play">Play</button>
```

Comment:
Looks like the play button doesn't do anything yet either. Leaving inline comments would have helped us  It's get an overall idea for the noise maker.